examples/pipelines/kritis-resources.yaml

```diff
@@ -1,26 +1,39 @@
 apiVersion: pipeline.knative.dev/v1alpha1
 kind: Resource
 metadata:
-  name: kritis-resources-sample
+  name: kritis-resources-git
   namespace: default
 spec:
-  resources:
-  - name: kritis-app-github-resource
-    type: git
-    params:
-    - name: url
-      value: https://github.com/grafeas/kritis
-    - name: revision
-      value: master
-  - name: kritis-test-github-resource
-    type: git
-    params:
-    - name: revision
-      value: master
-    - name: url
-      value: https://github.com/grafeas/kritis-test  
-  - name: stagingRegistry-resource
-    type: image
-    params:
-    - name: url
-      value: gcr.io/staging-image
\ No newline at end of file
+  name: kritis-app-github-resource
+  type: git
+  params:
+  - name: url
+    value: https://github.com/grafeas/kritis
+  - name: revision
+    value: master
+---    
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: Resource
+metadata:
+  name: kritis-resources-test-git
+  namespace: default
+spec:    
+  name: kritis-test-github-resource
+  type: git
+  params:
+  - name: revision
+    value: master
+  - name: url
+    value: https://github.com/grafeas/kritis-test  
+---    
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: Resource
+metadata:
+  name: kritis-resources-image
```

Comment:
some of the resources in these examples I don't see referenced anywhere, maybe the references in the Pipelines need to be updated?

Reply:
I'll check them again, might have missed it